src/Mod/Sketcher/Gui/SketcherSettingsDisplay.ui

```diff
@@ -92,324 +231,236 @@ Supports all unit systems except 'US customary' and 'Building US/Euro'.</string>
         </property>
        </widget>
       </item>
-      <item row="0" column="0">
-       <widget class="QLabel" name="label_6">
-        <property name="minimumSize">
-         <size>
-          <width>182</width>
-          <height>0</height>
-         </size>
+      <item row="8" column="0">
+       <widget class="Gui::PrefCheckBox" name="checkBoxShowDimensionalName">
+        <property name="toolTip">
+         <string>If checked, displays the name on dimensional constraints (if exists).</string>
```

Comment:
I suggest including the "%N" and "%V" in this tooltip. When I was testing and deleted both format characters, I could't remember what they were again to get them back :/ .

Reply:
There is another tooltip on the text input field documenting the possible parameters and a placeholder text with the default format.